### PR TITLE
[ticket/16316] Fix breadcrumbs markup validation

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-16316

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16316

### CODE REVIEW

File: phpBB/styles/prosilver/template/navbar_header.html [hunk 1]
```diff
@@ -178,32 +178,41 @@
...
-				<!-- EVENT overall_header_navlink_append -->
-			<!-- END navlinks -->
-			<!-- EVENT overall_header_breadcrumb_append -->
+
+		{% EVENT overall_header_breadcrumbs_before %}
+
+		<li class="breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
+
+			{% if U_SITE_HOME %}
+				<span class="crumb" {{ MICRODATA }}><a href="{{ U_SITE_HOME }}" id="{{ U_SITE_HOME | e('html_attr') }}" itemtype="https://schema.org/Thing" itemscope itemprop="item" data-navbar-reference="home"><i class="icon fa-home fa-fw" aria-hidden="true"></i><span itemprop="name">{{ L_SITE_HOME }}</span></a><meta itemprop="position" content="{{ navlink_position }}{% set navlink_position = navlink_position + 1 %}" /></span>
```
Comment (marc1706) [inline phpBB/styles/prosilver/template/navbar_header.html]: Is it just me or can this be removed? `itemtype="https://schema.org/Thing" itemscope`
IMHO it defaults to that anyway.

**Comment by marc1706** [comment]
The weird thing is that the structured data testing tool says everything is fine while the rich results test complains about issues still.

**Comment by 3D-I** [comment]
Yes, as per https://schema.org/BreadcrumbList

Tests:
https://search.google.com/structured-data/testing-tool
https://search.google.com/test/rich-results

Test URL: 
https://proteus.phpbbstudio.com/search.php?search_id=active_topics
https://proteus.phpbbstudio.com/viewtopic.php?f=2&p=2#p2

**Comment by marc1706** [comment]
The confusing thing is that the rich results test gives different results depending on whether passing it a site or sending code to it. Looks good now though. :+1:

**Pull request merged**